pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -237,31 +239,86 @@ func (p *cadvisorStatsProvider) ListPodCPUAndMemoryStats(_ context.Context) ([]s
 }
 
 // ImageFsStats returns the stats of the filesystem for storing images.
-func (p *cadvisorStatsProvider) ImageFsStats(ctx context.Context) (*statsapi.FsStats, error) {
+func (p *cadvisorStatsProvider) ImageFsStats(ctx context.Context) (*statsapi.FsStats, *statsapi.FsStats, error) {
```

Comment:
same as above, named return values will make clearer which is which

Reply:
done.